src/main/java/com/example/fourchelin/common/filter/MemberAuthenticationFilter.java

```diff
@@ -36,7 +36,7 @@ protected void doFilterInternal(HttpServletRequest request, HttpServletResponse
         if (session == null) {
             if(requestURI.startsWith("/api/searches")){ // /api/searches 로 시작하는 API는 익명 사용자도 접근이 가능
                 chain.doFilter(request, response);
-
+                return;
```

Comment:
익명 사용자도 접근이 가능하게 되었지만 이후에 세션에서 맴버를 가져오는 부분을 거치게 되어 
java.lang.NullPointerException: Cannot invoke "jakarta.servlet.http.HttpSession.getId()" because "session" is null
다시 한 번 에러를 발생하게 됩니다.
익명의 사용자는 이후에 맴버를 가져올 필요도 없기 때문에 바로 return해주어서 이후 로직을 거치지 않도록 하기 위해 설정했는데 혹시 다른 문제가 생기는 부분이 있을까요? ㅠㅠ